### ISSUE TITLE
Auto detected faces after image upload are calculated on original image after downscale

### ISSUE THREAD

**Issue by fschroiff** [mentosmenno2/image-crop-positioner#55]
Images that are too large for wordpress (that WP automatically downscales) have their spots calculated ont he original too large image instead of the scaled one.

```html
<div class="image-spots-preview">
	<img class="image-spots-preview__image" id="image-crop-positioner-image-spots-preview-image" src="https://rotterdam-innovation-city-acc.level-level.nl/wp-content/uploads/2022/04/albert-dera-ILip77SbmOE-unsplash-scaled.jpg?image-crop-positioner-ts=1648818047">
	<div class="image-spots-preview__spots"><div class="spot spot__face" data-x="304.2" data-y="304.2" data-width="3112.2" data-height="3112.2" style="top: 11.8828%; left: 17.8207%; width: 182.32%; height: 121.57%;"></div></div>
</div>
```

<img width="1242" alt="Screenshot 2022-04-01 at 15 22 18" src="https://user-images.githubusercontent.com/660010/161272032-4b5a1b02-39ae-4b06-95a1-0a97cd522257.png">

This is the image I uploaded: 
![albert-dera-ILip77SbmOE-unsplash (1)](https://user-images.githubusercontent.com/660010/161272511-7cb587a9-5c7c-4272-bc9a-957dd30295e6.jpeg)

**Comment by mentosmenno2** [comment]
Hi there @fschroiff , thanks for reporting your issue.
I was able to reproduce the issue and resolve it.

The pr is here: https://github.com/mentosmenno2/image-crop-positioner/pull/56

Will be merged soon in the next release.

**Comment by mentosmenno2** [comment]
This PR will be available in version v0.2.3
